src/views/App/index.tsx

```diff
@@ -27,6 +28,11 @@ class App extends React.Component<MainProps, MainState> {
               <Route path="/" exact component={Home} />
               <Route path="/login" exact component={Login} />
               <Route path="/sandbox" exact component={Sandbox} />
+              <SecureRoute
```

Comment:
Given this is information, I wonder if this actually needs to be a secure route. If it does, we need to update its syntax to match other secure routes.